doc/api/readline.md

```diff
@@ -447,6 +447,10 @@ autocompletion is disabled when copy-pasted input is detected.
 
 If the `stream` is a [TTY][], then it must be in raw mode.
 
+*Note*: This is automatically called by any readline instance on its `input`
+if the `input` is a terminal. Closing the `readline` instance does not stop
```

Comment:
Well … what I remember about these parts of the code mostly comes from reviewing ae178838ebde5093f316a827e74ba91b67df252d, and that was a year ago. I guess at this point we don’t have any real readline experts…